db/migrate/20241120130043_add_column_youth_court_fee_claimed_to_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnYouthCourtFeeClaimedToClaims < ActiveRecord::Migration[8.0]
+  def change
+    add_column :claims, :youth_court_fee_claimed, :string
```

Comment:
So at the moment NSM has three ways of indicating boolean fields. Some are strings with "yes"/"no" values, some are strings with "true"/"false" values, and some are actual booleans.

They should really all be booleans.

I would recommend using an actual boolean here as it's the most appropriate data type, but equally there are so many "yes"/"no" strings that if you want to go with that for maximal consistency I think it's fair enough.

Reply:
actually changed my mind - things aren't consistent enough that going one way or the other = more consistency so going with the more appropriate type